### DISCUSSION TITLE
Adding react-native-windows to an existing react-native project?

### DISCUSSION THREAD

**Discussion by jkoutavas** [microsoft/react-native-windows#11637] [Q&A] [answered]
I have an existing React Native app that builds for iOS, Android, and Web (react-native-web). Does someone have a recipe for adding react-native-windows to the mix? I'm brand new to react-native-windows.

**Comment by chrisglein** [comment, marked as answer]
Have you looked through the [Getting Started docs](https://microsoft.github.io/react-native-windows/docs/getting-started#install-react-native-for-windows)? They should cover this scenario. Let us know if you have any questions.

**Comment by jkoutavas** [comment]
Ahh, I missed the —overwrite flag. Thanks. Sent from my iPhoneOn May 22, 2023, at 2:27 PM, Chris Glein ***@***.***> wrote:﻿
Have you looked through the Getting Started docs? They should cover this scenario. Let us know if you have any questions.
